components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -306,7 +306,7 @@ function matchFunctions.getOpponents(args)
 	end
 
 	-- see if match should actually be finished if score is set
-	if isScoreSet and not Logic.readBool(args.finished) then
+	if isScoreSet and Logic.readBoolOrNil(args.finished) == nil then
```

Comment:
Would this not go wrong when `finished = true`?

Reply:
if it is set to false/0 we do not want it finished (that is the wanted change)

if it was set to true/1:
* `Logic.readBoolOrNil('true') == nil` is `false`
* `not Logic.readBool('true')` is `false` too
so no change here
(in line 321 we use `Logic.readBool(args.finished)` again before entering the match placement calculations, the above change is only in the if that confines the calculation against the time limit)

Reply:
I think it would be better to implement the `autofinished` param as suggested in Discord:

> Perhaps add a parameter autofinished that defaults to true but if set to false disables the functionality?

Reply:
done